deepmd/pt/utils/dataset.py

```diff
@@ -40,6 +43,38 @@ def __getitem__(self, index):
         b_data["natoms"] = self._natoms_vec
         return b_data
 
+    def get_frame_index(self):
```

Comment:
maybe `get_frame_index_for_elements`

Reply:
Is it possible to make element_counts keys be element itself (e.g. H O)? It's not readable only seeing int keys in `element_counts`.